fragments/k8s/cdk/bundle.yaml

```diff
@@ -10,7 +10,7 @@ applications:
     charm: "kubernetes-control-plane"
     num_units: 2
     options:
-      channel: 1.31/edge
+      channel: 1.32/stable
```

Comment:
in `main` we always stay on edge... i'm find with you bumping this to 1.33/edge or 1.32/edge

Reply:
but these channels are updated by this [script] when the bundle is "built"

[script]: https://github.com/charmed-kubernetes/bundle/blob/main/bundle